src/compute-client/src/protocol/response.proto

```diff
@@ -107,6 +107,7 @@ message ProtoSubscribeBatch {
 message ProtoStatusResponse {
   oneof kind {
     ProtoOperatorHydrationStatus operator_hydration = 1;
+    ProtoDataflowLimitStatus dataflow_limit_status = 2;
```

Comment:
I think this field should be called `dataflow_limit_exceeded`, since that's what the `StatusResponse` variant is called too?